package.json

```diff
@@ -63,14 +63,15 @@
     "react-addons-shallow-compare": "0.14.x - 15.x"
   },
   "scripts": {
+    "build-clean": "rm -rf dist && mkdir dist",
     "build-js": "babel src -d dist --copy-files",
     "build-js-example": "npm run build-js -- --source-maps inline",
     "build-docs": "(node-sass src/example/main.scss example.css) & (browserify -t brfs ./dist/example/main.js > example.js)",
-    "build-css": "node-sass src/main.scss main.css",
```

Comment:
I only changed the path of the compiled main css file for it to be inside the `dist` folder instead